bootstrap/src/index.ts

```diff
@@ -25,10 +20,12 @@ const skipOnError = (middleware: Middleware) => async (execute: ExecuteWrappedRe
 }
 
 // Make sure data has the same statusCode as the one we got as a result
-const withStatusCode = (execute: ExecuteWrappedResponse) => async (data_: AdapterRequest) => {
-  const { statusCode, data } = await execute(data_)
+// Is this still neccessary?
+const withStatusCode: Middleware = (execute) => async (data_: AdapterRequest) => {
+  const { statusCode, data, ...rest } = await execute(data_)
```

Comment:
What is the reason you introduced this extra `...rest` data to be outputted?

Reply:
`AdapterResponse` has a required JobRunID
```
  export type AdapterResponse = {
    jobRunID: string
    statusCode: number
    data: any
    result?: any
  }
```